CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Colaborando
+
+É possível colaborar de duas formas, coletando dados divulgados pelas Sec. de Saúde dos Estados, ou participando do projeto no Github, propondo melhorias, revisando código, discutindo nas issues e afins.
+
+## Coletando Dados
+
+Para participar do time de coleta de dados, acesse [este link](https://go.rocket.chat/invite?host=chat.brasil.io&path=invite%2FMiMMCk) faça o cadastro e forneça as informações solicitadas no canal `#covid19-onboarding`.
+
+### Canais de comunicação
+
+#### Gerais
+
+Existem dois canais que todos os voluntários devem fazer parte, são eles :
+
+- #covid19
+- #covid19-anuncios
+
+O #covid19 serve para discussão de duvidas gerais entre todos os voluntários, e o #covid19-anuncios serve para anuncios dos administradores do projeto sobre atualização de dados e outros informes importantes, e é fechado para conversas.
+
+#### Regionais
+
+Para coordençao dos trabalhos de colete e checagem, você deve então acessar os canais correspondentes à região dos estados escolhidos, são eles :
+
+- #covid19-centro-oeste
+- #covid19-nordeste
+- #covid19-norte
+- #covid19-sudeste
+- #covid19-sul
+
+Haverá ao menos um coordenador por região, que deve ficar atento às atualizações dos estados, para manter as planilhas sempre atualizadas assim que os boletins forem divulgados, e cobrar as checagens dos dados.
+
+### Planilhas
+
+Os voluntários receberão acesso às planilhas para lançamento e checagem dos dados.
+
+Existe uma planilha para cada estado, nesta planilha existem dois pares de abas, como segue :
+
+- Boletins [RASCUNHO]
+- Casos [RASCUNHO]
+- Boletins [FINAL]
+- Casos [FINAL]
+
+Cada voluntário deve fazer o preenchido ou a checagem.
+
+O preenchimento deve sempre ser feito nas abas de sufixo `[RASCUNHO]`, na aba `Boletins [RASCUNHO]` devem ser informados, para cada boletim com informações atualizadas, uma linha com a data do boletim e o link para acesso àquele boletim. Com os dados daquele boletim, deve ser preenchida a aba `Casos [RASCUNHO]`, criando um par de colunas para a data correspondente com os valores de casos `confirmados` e `mortes`, detalhando por município.
+
+No processo de checagem, o voluntário deve verificar nos links informados na planilha `Boletins [RASCUNHO]` se os dados informados no boletim correspondem aos dados preenchidos na planilha `Casos [RASCUNHO]`, caso esteja tudo OK, basta copiar os dados das planilhas de sufixo `[RASCUNHO]` para a planilha correspondente com sufixo `[FINAL]`.
```

Comment:
Acho que vale adicionar que esse processo está mais detalhado na metodologia (pode adicionar e fazer o commit direto no `master`, beleza?).

Reply:
ok, o que for de README.me, que não gere impacto, farei commit direto